tests/integration/test_compute.py

```diff
@@ -0,0 +1,70 @@
+from fastapi import params
+import numpy as np
+import pandas as pd
+
+
+OBSERVATION = (
+    "FOPR",
+    {"values": [1, 2, 3], "errors": [0.1, 0.2, 0.3], "x_axis": ["C", "E", "H"]},
+)
+
+
+def test_univariate_misfits_with_labels(client, create_experiment, create_ensemble):
```

Comment:
I think it also would be nice to test that the actual misfit calculation is correct :)

Reply:
I thought about it, but that would require to have a fix matrix and not a random one, which is doable of-course

Reply:
That is fine, I think we should validate our "compute" results with tests :)

Reply:
I did the computations test also. Is this ok?

Reply:
Pre-computed data available as realized it won't be super easy to do it algorithmically.